http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpWdsDAOSpec.scala

```diff
@@ -99,11 +99,6 @@ class HttpWdsDAOSpec extends AnyFlatSpec with Matchers with LeonardoTestSuite wi
     val cosStatus = coaRes.unsafeRunSync()
     cosStatus shouldBe true
 
-    // Status fails when we wrongly put apptype as WDS
-    val coaResBad = coaWdsDAO.getStatus(Uri.unsafeFromString(baseUri), authHeader, AppType.Wds)
```

Comment:
This test might need to be modified more?  I'm not exactly sure how it's working but it looks like it's examining the WDS in the combined app so either it should be removed entirely/replaced with a WDS-only test, or otherwise modified.

Reply:
since wds will no longer be part of cromwell, there should be only one standalone wds which the other 2 tests should cover - or am I misunderstanding something?